python/minimal_connection.py

```diff
@@ -33,28 +35,38 @@ def main():
     # Parse the command-line.
     args = parser.parse_args()
 
-    githash = "next"
```

Comment:
I changed the default from next to using the XML in tree, which seemed more sensible.  Willing to go back.